models/connector.go

```diff
@@ -1,20 +1,25 @@
 package models
 
+// Connector Struct representing a connector from the config
 type Connector struct {
-	Type     string
-	ID       string
-	Tags     string
-	Active   bool
-	Server   string
-	Port     string
-	Login    string
-	Pass     string
-	File     string
-	From     string
-	Key      string
-	Image    string
-	Users    string
-	Commands []Command
-	Debug    bool
-	BotName  string
+	Type              string
+	ID                string
+	Tags              string
+	Active            bool
+	Server            string
+	Port              string
+	Login             string
+	Pass              string
+	File              string
+	From              string
+	Key               string
+	Secret            string
+	AccessToken       string
+	AccessTokenSecret string
+	Image             string
```

Comment:
I understand that twitter requires two sets of keys, but how do we better label these to differentiate them from key/secret. Or, should we create a more generic array of key/values?